The format specifier for size_t is %zu and not %lu. Fixing this to
prevent issues on 32 bit machines.

--- AFLplusplus-4.00c/instrumentation/SanitizerCoverageLTO.so.cc
+++ AFLplusplus-4.00c/instrumentation/SanitizerCoverageLTO.so.cc
@@ -1080,7 +1080,7 @@
       }
 
       if (!be_quiet)
-        printf("AUTODICTIONARY: %lu string%s found\n", count,
+        printf("AUTODICTIONARY: %zu string%s found\n", count,
                count == 1 ? "" : "s");
 
       if (count) {
